<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4" stretch="14,2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_UP" stretch="2,3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_LL" stretch="10,2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_CONTROL">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_LEFT" stretch="25,1,12,1,23">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <widget class="QGroupBox" name="groupBoxParam">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(251, 255, 212);
color: rgb(0, 0, 0);</string>
              </property>
              <property name="title">
               <string>分割参数调节</string>
              </property>
              <widget class="QWidget" name="layoutWidgetSegParam">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>20</y>
                 <width>181</width>
                 <height>181</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_SegParam">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_Stride">
                  <item>
                   <widget class="QLabel" name="labelStride">
                    <property name="text">
                     <string>推理窗口步长</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBoxStride">
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="maximum">
                     <number>256</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_Confidence">
                  <item>
                   <widget class="QLabel" name="labelConfidence">
                    <property name="text">
                     <string>置信度</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="doubleSpinBoxConfidence">
                    <property name="minimum">
                     <double>0.000000000000000</double>
                    </property>
                    <property name="maximum">
                     <double>1.000000000000000</double>
                    </property>
                    <property name="singleStep">
                     <double>0.010000000000000</double>
                    </property>
                    <property name="value">
                     <double>0.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QRadioButton" name="radioButtonUsingGPU">
                  <property name="text">
                   <string>使用GPU</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBoxSegPostProcess">
                  <property name="text">
                   <string>分割结果后处理</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBoxPostProcess">
              <property name="title">
               <string>后处理参数调节</string>
              </property>
              <widget class="QWidget" name="layoutWidgetPostProcess">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>20</y>
                 <width>181</width>
                 <height>81</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_Circularity">
                  <item>
                   <widget class="QLabel" name="labelCircularity">
                    <property name="text">
                     <string>圆形度过滤</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="doubleSpinBoxCircularity">
                    <property name="maximum">
                     <double>1.000000000000000</double>
                    </property>
                    <property name="singleStep">
                     <double>0.010000000000000</double>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_Area">
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>面积过滤</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBoxArea">
                    <property name="maximum">
                     <number>9999</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBoxBatchParam">
              <property name="title">
               <string>批量输出控制</string>
              </property>
              <widget class="QWidget" name="layoutWidgetBatchOutput">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>20</y>
                 <width>181</width>
                 <height>191</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_BatchOutputControl">
                <item>
                 <widget class="QCheckBox" name="checkBoxSaveSeg">
                  <property name="text">
                   <string>保存分割结果</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBoxSavePredict">
                  <property name="text">
                   <string>保存原图检测结果</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBoxGenCSV">
                  <property name="text">
                   <string>导出详细记录</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBoxGenExcel">
                  <property name="text">
                   <string>导出板式分析结果</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_RIGHT" stretch="5,1,6,1,5">
            <item>
             <widget class="QGroupBox" name="groupBoxClass">
              <property name="title">
               <string>检测参数调节</string>
              </property>
              <widget class="QWidget" name="classParamContainer">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>20</y>
                 <width>161</width>
                 <height>101</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_classConfidience">
                  <item>
                   <widget class="QLabel" name="labelClassConfidience">
                    <property name="text">
                     <string>置信度</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="doubleSpinBoxclassConfidience"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QRadioButton" name="radioButtonClassUsingGPU">
                  <property name="text">
                   <string>使用GPU</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBoxSingle">
              <property name="title">
               <string>单张图片操作</string>
              </property>
              <widget class="QWidget" name="layoutWidgetSingle">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>20</y>
                 <width>141</width>
                 <height>141</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_Single">
                <item>
                 <widget class="QPushButton" name="pushButtonSingleImage">
                  <property name="text">
                   <string>选择图片</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButtonSegmentation">
                  <property name="text">
                   <string>板孔位置分割</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButtonSegPostProcess">
                  <property name="text">
                   <string>后处理</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButtonROI">
                  <property name="text">
                   <string>ROI裁切</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButtonPositivePredict">
                  <property name="text">
                   <string>阳性孔预测</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBoxBatch">
              <property name="title">
               <string>批量操作</string>
              </property>
              <widget class="QWidget" name="layoutWidgetBatch">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>20</y>
                 <width>151</width>
                 <height>121</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_Batch">
                <item>
                 <widget class="QPushButton" name="pushButtonMutiImage">
                  <property name="text">
                   <string>批量选择图片</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButtonSavepath">
                  <property name="text">
                   <string>选择保存路径</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButtonStartBatch">
                  <property name="text">
                   <string>开始批量分割</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButtonStartBatchPredict">
                  <property name="text">
                   <string>开始批量检测</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowserLog"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_WorkSpace">
        <item>
         <widget class="QGraphicsView" name="graphicsViewInput">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(206, 255, 235);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicsViewOutput">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(215, 240, 255);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widgetBottom" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 127);</string>
      </property>
      <widget class="QLabel" name="labelModel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>241</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>当前选择模型：</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelWeight">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>10</y>
         <width>651</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>模型权重路径：</string>
       </property>
      </widget>
      <widget class="QWidget" name="bottomProgressContainer">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>1001</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,1">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_PatchProgress">
          <item>
           <widget class="QLabel" name="labelPatchProgress">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>单张分割进度</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBarPatch">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_ClassProgress">
          <item>
           <widget class="QLabel" name="labelClassProgress">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>单张分类进度</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBarClass">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_BatchProgress">
          <item>
           <widget class="QLabel" name="labelBatchProgress">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>批量检测进度</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBarBatch">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>模型选择</string>
    </property>
    <addaction name="actionUnet"/>
    <addaction name="actionAttentionUnet"/>
    <addaction name="actionR2Unet"/>
    <addaction name="actionFCN8S"/>
    <addaction name="actionSwinUnet"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>权重选择</string>
    </property>
    <addaction name="actionUnet_weight"/>
    <addaction name="actionAttentionUnet_weight"/>
    <addaction name="actionR2Unet_weight"/>
    <addaction name="actionFCN8S_weight"/>
    <addaction name="actionSwinUnet_weight"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="actionUnet">
   <property name="text">
    <string>Unet</string>
   </property>
  </action>
  <action name="actionAttentionUnet">
   <property name="text">
    <string>AttentionUnet</string>
   </property>
  </action>
  <action name="actionUnet_weight">
   <property name="text">
    <string>Unet</string>
   </property>
  </action>
  <action name="actionAttentionUnet_weight">
   <property name="text">
    <string>AttentionUnet</string>
   </property>
  </action>
  <action name="actionR2Unet">
   <property name="text">
    <string>R2Unet</string>
   </property>
  </action>
  <action name="actionFCN8S">
   <property name="text">
    <string>FCN8S</string>
   </property>
  </action>
  <action name="actionSwinUnet">
   <property name="text">
    <string>SwinUnet</string>
   </property>
  </action>
  <action name="actionR2Unet_weight">
   <property name="text">
    <string>R2Unet</string>
   </property>
  </action>
  <action name="actionFCN8S_weight">
   <property name="text">
    <string>FCN8S</string>
   </property>
  </action>
  <action name="actionSwinUnet_weight">
   <property name="text">
    <string>SwinUnet</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
